### ISSUE TITLE
Problems from the start

### ISSUE THREAD

**Issue by anderscl** [usnistgov/REFPROP-issues#305]
If you have an issue to report, please continue and fill out the applicable sections below.  The details provided will help to resolve this issue as quickly as possible.

### Prerequisites

**NOTE**: Be aware that all issues are publicly accessible and viewable.  Thus please do not post any code or other content that is protected intellectual property or under copyright.

Before posting an issue, please:  
    * Check the [FAQ](https://pages.nist.gov/REFPROP-docs/) for common solutions.  
    * Check that your issue isn't already filed under [REFPROP Issues](https://github.com/usnistgov/REFPROP-issues/).  
    * Check the [documentation](http://refprop-docs.readthedocs.io/en/latest/).  

You can also use markdown to format your issue: [GitHub guide on Markdown](https://guides.github.com/features/mastering-markdown/). If you have code snippets, please use a code block to ensure that the formatting remains legible in the web interface.  For instance, surround your code in triple backticks:

``` python
print('Hello world!')
```

### Description

I downloaded REFPROP and put the folder where it suggested: c:\Program Files (x86)\Refprop. I followed the instructions and step 1 was see if the calcs worked in REFPROP.xls and yes, they did. But getting the DLL links working...I was stopped by too many errors to mention.

I sent an email to Eric and Ian and Eric suggested to first move the folder out of the C drive. I did that but now Step 1 is a bust. The output values have an error: Error in opening file for....

Am I beyond hope?

### Steps to Reproduce

1. [First Step]
2. [Second Step]
3. [and so on...]

**Expected behavior:** [What you expect to happen]

**Actual behavior:** [What actually happens]

### Versions

**REFPROP Version:** [REFPROP version you are using]  
**Operating System and Version:** [OS you are running REFPROP on]  
**Access Method:** [How you are accessing REFPROP (i.e. Python, other wrapper)]  

### Additional Information

If possible, please post examples and/or screenshots of the issue.

**Comment by ianhbell** [comment]
Personally, I never install REFPROP in the recommended location, and always install in a location where I have read/write ability, which for me is "D:/Program Files (x86)/REFPROP".  That avoids a family of weird behaviors caused by the security system of windows.

**Comment by anderscl** [comment]
OK   But when I moved the folder, I have more problems. Should I or can I
re-download and re-install without paying again?

Best Regards,


*Charlie Anderson*
(M) 302 981 4028

**Comment by ianhbell** [comment]
Yes, I would delete the folder you copied, uninstall REFPROP, and re-install.  The key point is that environment variable RPPREFIX is still pointing to your old folder, and the installer will fix that on re-install.

**Comment by anderscl** [comment]
OK    How to uninstall?  Just delete the folder?

For reinstall, just run NIST2310.exe again? and of course point to a D:
location?

Best Regards,


*Charlie Anderson*
(M) 302 981 4028

**Comment by ianhbell** [comment]
Yes, delete the folder, then run the uninstall option from windows, then run the NIST2310.exe again to install.

**Comment by EricLemmon** [comment]
I'm moving this issue over to the wrappers section so that others will find it when searching there.  The "issues" part of our github site is for everything not related to the use of the DLL, except the GUI.
